specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/ddosProtectionPlan.json

```diff
@@ -180,6 +180,58 @@
             "$ref": "./examples/DdosProtectionPlanCreate.json"
           }
         }
+      },
+      "patch": {
+        "tags": [
+          "DdosProtectionPlans"
+        ],
+        "operationId": "DdosProtectionPlans_UpdateTags",
+        "description": "Update a DDoS protection plan tags",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "ddosProtectionPlanName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the DDoS protection plan."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "./network.json#/definitions/TagsObject"
+            },
+            "description": "Parameters supplied to the update DDoS protection plan resource tags."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Update successful. The operation returns the resulting DDoS protection plan resource.",
+            "schema": {
+              "$ref": "#/definitions/DdosProtectionPlan"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
```

Comment:
>"x-ms-long-running-operatio [](start = 8, length = 27)

should have 201, similar to PUT, as it is marked as long running operation.. ('201 + provisioningState' pattern)

Reply:
Only changes in the Tags are supported with PATCH call. That's why, there won't be 201 for this call.

Reply:
In such a case, can you remove  "x-ms-long-running-operation": true, or set it to false

Reply:
changed to false